ts/de_DE.ts

```diff
@@ -317,7 +317,7 @@
     <message>
         <location filename="gui.py" line="1268"/>
         <source>In what units you want to scale?</source>
-        <translation>In welcher Einheit möchtest du skalieren?</translation>
+        <translation>In welcher Einheit möchten Sie skalieren?</translation>
```

Comment:
As a German i would love if everybody would use 'Du' but i think it is just in my 'blood' to use the formal addressing...It is just something we got BURNED in our brains. Guess no one will be offended if we use the 'Du' ... at least not in this community. :-) Mixing is really the worst...i will change it to 'Du'.